core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -1928,9 +1928,15 @@ class TaskSetManagerSuite
     // get the TaskSetManager
     val manager = sched.invokePrivate(taskSetManagers()).get(stageId).get(stageAttemptId)
 
-    val task0 = manager.resourceOffer(exec0, "localhost", TaskLocality.NO_PREF)
-    val task1 = manager.resourceOffer(exec1, "localhost", TaskLocality.NO_PREF)
-    assert(task0.isDefined && task1.isDefined)
+    var task0 : Option[TaskInfo] = None
```

Comment:
Nit: remove space before colon.
Can you just write:
```
val (task0, task1) = eventually(...) {
  ...
  (task0, task1)
}

Reply:
Done